diff --git a/dist/src/stryker-setup.js b/dist/src/stryker-setup.js
index aecfacbe47b285c272c47e54b3d66a716adfdc41..499c98102699f68d0134a97a54f1d7ba223e5b73 100644
--- a/dist/src/stryker-setup.js
+++ b/dist/src/stryker-setup.js
@@ -60,10 +60,12 @@ function collectTestName({ name, suite, }) {
     const nameParts = [name];
     let currentSuite = suite;
     while (currentSuite) {
-        nameParts.unshift(currentSuite.name);
+        if (currentSuite.name) {
+            nameParts.unshift(currentSuite.name);
+        }
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(' > ').trim();
 }
 function toRawTestId(test) {
     return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
diff --git a/dist/src/test-helpers.js b/dist/src/test-helpers.js
index 56cc786f30c524ae18ce87eef1a67fdf26b703c5..e7b344852c1ab677006855d9fb44e56b49f89030 100644
--- a/dist/src/test-helpers.js
+++ b/dist/src/test-helpers.js
@@ -5,10 +5,12 @@ export function collectTestName({ name, suite, }) {
     const nameParts = [name];
     let currentSuite = suite;
     while (currentSuite) {
-        nameParts.unshift(currentSuite.name);
+        if (currentSuite.name) {
+            nameParts.unshift(currentSuite.name);
+        }
         currentSuite = currentSuite.suite;
     }
-    return nameParts.join(' ').trim();
+    return nameParts.join(' > ').trim();
 }
 export function toRawTestId(test) {
     return `${test.file?.filepath ?? 'unknown.js'}#${collectTestName(test)}`;
